### PR TITLE
PC-2208 - Android Google Tag Manager documentation

### PR DESCRIPTION
**Pull request by radixdev**
Google Tag Manager integration guide and example code.

https://jira.braze.com/browse/PC-2208

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+nav_title: Google Tag Manager
+platform: Android
+page_order: 8
+
+---
+### Google Tag Manager
+
+#### Initializing the SDK {#initializing-android-google-tag-provider}
+
+Braze's SDK can be initialized and controlled within tags configured from Google Tag Manager.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md:11]: Link out to GTM homepage

**Comment by KellieHawks** [comment]
@cla-bot check

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md (hunk 1 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md:11]: done

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+Lastly, log an event through Firebase/Google Tag Manager using the Firebase Analytics event name `logEvent`. Note that `eventName` is the name of the Braze event being logged. All other keys present, such as `My Boolean Parameter` will be added as event properties to the logged event.
+
+{% tabs %}
+{% tab JAVA %}
+
+```java
+Bundle params = new Bundle();
+params.putString("eventName", "played song event");
+params.putBoolean("My Boolean Parameter", true);
+params.putString("My String Parameter", "hello");
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/android_google_tag_manager.md:314]: `params.putString("genre", "pop music");`

**Comment by davidbielik** [comment]
hey @KellieHawks ready to merge, thanks!

**Pull request merged**